saharspec.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
     |\.rubocop.yml
     |\.travis.yml
     )$/x
-  end
+  end + %w[config/rubocop-rspec.yml]
```

Comment:
This statement is redundant. See how the initial list of the files was created:
```ruby
`git ls-files`.split($RS)
```

Once PR will be accepted, your `config/rubocop-rspec.yml` will be included automatically.